include/pfs/types.hpp

```diff
@@ -604,6 +604,22 @@ struct id_map
     size_t length = UINT32_MAX;
 };
 
+// Based on https://elixir.bootlin.com/linux/v5.19.10/source/net/ipv4/fib_trie.c#L2998
+struct route
+{
+    std::string iface;
+    ip destination;
+    ip gateway;
+    unsigned int flags;
+    unsigned short refcnt;
+    unsigned short use;
+    uint32_t metric;
```

Comment:
I think that even though the struct has `uint32_t fib_priority`, because the `printf` uses `%d`, it's going to be more consistent to define that as `int`. wdyt?